platform/linux-generic/odp_packet.c

```diff
@@ -1567,10 +1567,11 @@ void odp_packet_print(odp_packet_t pkt)
 	int len = 0;
 	int n = max_len - 1;
 	odp_packet_hdr_t *hdr = packet_hdr(pkt);
-	odp_buffer_t buf      = packet_to_buffer(pkt);
 
-	len += snprintf(&str[len], n - len, "Packet ");
-	len += odp_buffer_snprint(&str[len], n - len, buf);
+	len += snprintf(&str[len], n - len, "Packet\n------\n");
```

Comment:
odp_buffer_print() doesn't print similar underlining. Is this intentional?

Reply:
No, I'll add it there as well. I added it into this function as the other packet print had it.